src/tokens/blockchain.service.ts

```diff
@@ -43,6 +43,12 @@ export class BlockchainConnectorService {
   password: string;
   passthroughHeaders: string[];
 
+  retryBackOffFactor: number;
+  retryBackOffLimit: number;
+  retryBackOffInitial: number;
+  retryCondition: string;
+  retriesMax: number;
```

Comment:
We've ended up with quite a lot of member variables... think it might be cleaner to introduce a single `options` object here to contain them all? It would allow the `configure()` method below to just take in and set a single variable.